packages/E2ETest/wdio/test/DirectManipulation.spec.ts

```diff
@@ -28,6 +31,9 @@ describe('DirectManipulationTest', () => {
       result.includes('width=50'),
       'measureLayout response has correct width'
     );
-    assert.ok(result.includes('x=20;'), 'measureLayout response x=20');
+    assert.ok(
+      result.includes('x=20;'),
+      'measureLayout response x!=20, result: ' + result
+    );
```

Comment:
consider making this change in a separate PR as it isn't related to VS/142 :)

Reply:
it's the only test that catches that vs2019 has bugs